ACKategoriesCore/StringExtensions.swift

```diff
@@ -18,9 +18,14 @@ extension String {
         return !isEmpty && rangeOfCharacter(from: CharacterSet.decimalDigits.inverted) == nil
     }
 
-    /// Uses self as key to Localizable.strings and returns its localized value or self
-    public func localized() -> String {
-        return NSLocalizedString(self, comment: "")
+    /// Uses `self` as key in stringsfile and returns its localized value or `self`
+    ///
+    /// - parameter comment: Comment to string usage, is displayed if `key` is missing in table and `value` is empty string
+    /// - parameter value: Value to be displayed if `key` is missing, if empty string then `comment` is displayed
```

Comment:
Is this true? [Official documentation](https://developer.apple.com/documentation/foundation/1418095-nslocalizedstring)  says nothing about that, but there's companion [ObjC func](https://developer.apple.com/documentation/foundation/nslocalizedstringwithdefaultvalue) and it does not seem to work as described here.
If value is not defined AND key is missing in .strings, I think key is used, isn't it..? Comment is just a comment and is never used for anything...

Reply:
If you open this function in Xcode, you will see hundreds of lines of documentation that start with this.

```
/// Returns the localized version of a string.
///
/// - parameter key: An identifying value used to reference a localized string.
///   Don't use the empty string as a key. Values keyed by the empty string will
///   not be localized.
/// - parameter tableName: The name of the table containing the localized string
///   identified by `key`. This is the prefix of the strings file—a file with
///   the `.strings` extension—containing the localized values. If `tableName`
///   is `nil` or the empty string, the `Localizable` table is used.
/// - parameter bundle: The bundle containing the table's strings file. The main
///   bundle is used by default.
/// - parameter value: A user-visible string to return when the localized string
///   for `key` cannot be found in the table. If `value` is the empty string,
///   `key` would be returned instead.
/// - parameter comment: A note to the translator describing the context where
///   the localized string is presented to the user.
///
/// - returns: A localized version of the string designated by `key` in the
///   table identified by `tableName`. If the localized string for `key` cannot
///   be found within the table, `value` is returned. However, `key` is returned
///   instead when `value` is the empty string.
```

Don't know why this is not displayed on web.

Reply:
Well... and it says the same as I do 😄 

```suggestion
    /// - parameter value: Value to be displayed if `key` is missing, if empty string then `key` is displayed
```